Test Automation Course./src/topic8/ServiciuStoc.java

```diff
@@ -0,0 +1,21 @@
+package topic8;
+
+public class ServiciuStoc {
+    public static void main(String[] args) {
```

Comment:
pentru a practica mai bine lucrul cu colectii as sugera sa creezi toate aceste obiecte la fel intr-o colectie :

List<Carte> carti = new ArrayList<>();
sa faci un loop in care adaugi toate aceste valori

Reply:
ok.multumesc.